examples/FCCee/higgs/mH-recoil/histmaker.py

```diff
@@ -0,0 +1,181 @@
+
+# list of processes (mandatory)
+processList = {
+    #'p8_ee_ZZ_ecm240':{'fraction':1},
+    #'p8_ee_WW_ecm240':{'fraction':1}, 
+    #'wzp6_ee_mumuH_ecm240':{'fraction':1},
+    'p8_ee_WW_ecm240':{'fraction':1, 'input': 'localSamples/p8_ee_WW_ecm240/', 'crossSection': 0.25792}, #   16.4385
+    'p8_ee_ZZ_ecm240':{'fraction':1, 'input': 'localSamples/p8_ee_ZZ_ecm240/', 'crossSection': 1.35899},
+    'p8_ee_ZH_ecm240':{'fraction':1, 'input': 'localSamples/p8_ee_ZH_ecm240/', 'crossSection': 0.0067643},  # 0.201868
```

Comment:
is p8_ee_ZH_ecm240 part of the winter2023/IDEA campaign?

Reply:
No, it isn't. So in case the process does not exist, the user can provide a custom cross-section in the processList. Otherwise, a default of 1 is used